### DISCUSSION TITLE
Configure lifecycle to S3 in distributed mode

### DISCUSSION THREAD

**Discussion by liranblum** [minio/minio#20556] [Q&A]
Hello,
I’m trying to configure lifecyle (ilm) from minio to S3 using ilm add tier however I get This 'admin' API is not supported by server in 'mode-server-distributed-xl'. as my configuration is indeed distributed.
How would it be possible to have a transition to S3 after x days, in distributed mode?
Thank you

**Comment by klauspost** [comment]
Check your mc version is the same or newer than the MinIO server you are running. (Or even better upgrade both to latest)

Without you providing any real information that is my best guess.
